integrations/terraform-cloud/main.py

```diff
@@ -165,8 +164,21 @@ async def resync_state_versions(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
         yield enriched_state_versions_batch
 
 
+@ocean.on_resync()
+async def on_create_webhook_resync(kind: str) -> RAW_RESULT:
+    terraform_client = init_terraform_client()
+    if app_host := ocean.integration_config.get("app_host"):
+        await terraform_client.create_workspace_webhook(app_host=app_host)
+    return []
+
+
 @ocean.router.post("/webhook")
 async def handle_webhook_request(data: dict[str, Any]) -> dict[str, Any]:
+    print(data)
+    for notifications in data["notifications"]:
+        if notifications["trigger"] == "verification":
```

Comment:
You can also add a logger.info here that the verification request challenge has been accepted